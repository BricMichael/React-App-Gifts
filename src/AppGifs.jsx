import React, { useState } from 'react';
import AddCategories from './components/AddCategories';
import GifGrid from './components/GifGrid';


const AppGifs = () => {

     const [ categories, setCategories ] = useState(['Perfect']);
    

    return(
        <>
            <h1>APPGIFS</h1>
            <AddCategories setCategories = { setCategories } />
            <br/>
            <ol>
               {
                   categories.map( item => (

                        <GifGrid 
                         key = { item }
                         category = { item }                                              
                        />
                        )
                    )
               }         
            </ol>
        </>
    ); 
}

export default AppGifs;