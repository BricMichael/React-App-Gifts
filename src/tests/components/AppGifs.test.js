import React from 'react';
import { shallow } from "enzyme";
import AppGifs from "../../AppGifs";



describe('Pruebas en el componente <AppGifs />', () => {
    const wrapper = shallow( <AppGifs /> )

    test('Mostar el componente', () => {

        expect( wrapper ).toMatchSnapshot()

    })

})


