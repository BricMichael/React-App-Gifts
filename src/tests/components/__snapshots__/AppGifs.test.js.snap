// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente <AppGifs /> Mostar el componente 1`] = `
<Fragment>
  <h1>
    APPGIFS
  </h1>
  <AddCategories
    setCategories={[Function]}
  />
  <br />
  <ol>
    <GifGrid
      category="Perfect"
      key="Perfect"
    />
  </ol>
</Fragment>
`;
